source/_integrations/fireservicerota.markdown

```diff
@@ -0,0 +1,286 @@
+              {{ states('sensor.incidents_speech') }}
+          media_content_type: 'audio/mp4'
+      - service: homeassistant.turn_on
+        data_template: 
+          entity_id: >
+              light.bwr_nesthub
+      - service: timer.start
+        data:
+          entity_id: timer.bwr_nesthub
+    condition:
+      condition: and
+      conditions:
+      - condition: time
+        after: '06:30:00'
+        before: '23:00:00'
+      - condition: template
+        value_template: "{{ not is_state('sensor.incidents_speech', 'None') }}"
+      - condition: state
+        entity_id: input_boolean.bwr_tts
+        state: 'on'
+
+  - alias: Turn off Nest Hub screen 2 minutes after trigger
+    trigger:
+      platform: event
+      event_type: timer.finished
+      event_data:
+        entity_id: timer.bwr_nesthub
+    action:
+      service: light.turn_off
+      data:
+        entity_id:
+          - light.bwr_nesthub
+
+  - alias: Cast to Nest Hub
+    trigger: 
+      platform: homeassistant
+      event: start
+    action:
+      service: cast.show_lovelace_view
+      data: 
+        entity_id: media_player.nest_hub_slaapkamer
+        view_path: bwr
+
+timer:
+  bwr_nesthub:
+    duration: '00:02:00'
+    name: Nest Hub Screen Timer
+```
+
+{% endraw %}
+
+Example Lovelace Dashboard
+
+```yaml
+title: BWR
+path: bwr
+panel: true
+icon: mdi:fire-truck
+cards:
+  - type: 'custom:layout-card'
+    cards:
+      - type: 'custom:bignumber-card'
+        entity: sensor.incidents
+        scale: 15px
+      - type: map
+        entities:
+          - entity: sensor.incidents
+        hours_to_show: 0
+        default_zoom: 15
+      - type: 'custom:vertical-stack-in-card'
+        cards:
+          - type: history-graph
+            entities:
+              - entity: sensor.incidents
+            hours_to_show: 1
+            refresh_interval: 0
+          - type: 'custom:home-feed-card'
+            show_empty: false
+            entities:
+              - entity: sensor.incidents
+                max_history: 3
+                include_history: true
+          - type: entities
+            show_header_toggle: false
+            entities:
+              - input_boolean.bwr_tts
+              - light.bwr_nesthub
+          - type: 'custom:bignumber-card'
+            entity: sensor.time
+            scale: 25px
+```
+
+
+You can cast this dashboard to a Nest Hub or other ChromeCast capable device using a Nabu Casa subscription or 'Catt to Cast'.
+
+Below is the result of above code, an audio player via 'browser_mod' player play text to speech data, the screen gets enabled when an incident arrives and goes dark after 2 minutes. You can toggle TTS and screen display on/off.
+
+I have muted the announcements outside 23:00-6:30h using a automation condition, if you are on duty that's something to remove/change.
+
+### Example
+
+<p class='img'>
+  <img src='{{site_root}}/images/integrations/fireservicerota/dashboard.png' />
```

Comment:
Do not use `{{site_root}}`, please add an alt text to keep our website accessible for those who can't see the image.

Reply:
```shell
laptop-ubuntu:~/development/home-assistant.io/source/_integrations$ grep site_root *|wc -l
55
````
Will do, there are 55 other markdown files still using it.

Reply:
Done